resources/dicts/patrols/plains/border/leaf-bare.json

```diff
@@ -1 +1,3556 @@
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_coyote_danger1",
+                "can_have_stat": ["p_l"],
+                "stat_trait":[
+                    "bloodthirsty",
+                    "confident",
+                    "fierce",
+                    "righteous",
+                    "arrogant",
+                    "ambitious",
+                    "daring",
+                    "bold"
+                ],
+                "injury": [
+                    {
+                        "cats": ["multi"],
+                        "injuries": ["big_bite_injury", "scrapes"],
+                        "scars": ["NECKBITE"]
+                    },
+                    {
+                        "cats": ["multi"],
+                        "injuries": ["scrapes"],
+                        "scars": []
+                    }
+                ],
+                "dead_cats":["multi"],
+                "history_text": {
+                    "scar": "Courting coyotes scarred r_c's pelt.",
+                    "reg_death": "r_c died from injuries sustained keeping coyotes away from c_n's borders.",
+                    "lead_death": "from injuries sustained keeping coyotes away from c_n's borders"
+                },
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["respect", "trust"],
+                        "amount": -10
+                    },
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["clan"],
+                        "mutual": false,
+                        "values": ["respect"],
+                        "amount": -10
+                    }
+                ]
+            },
+            {
+                "text": "That yip seemed a little weird. Ears pricked, the patrol stalks closer, crossing the border. On the other side, a female assesses the males trying to court her. None notice the cats, but the patrol looks to each other with dismay. The patrol tries to lead the pack away with a relay of cats distracting them, but r_c gets caught by a male's jaws, and it's only s_c's first aid that saves {PRONOUN/r_c/object} from dying in the snow.",
+                "exp": 0,
+                "weight": 40,
+                "art": "fst_hunt_howl",
+                "can_have_stat": ["not_pl_rc"],
+                "stat_skill": ["HEALER,0"],
+                "injury": [
+                    {
+                        "cats": ["r_c"],
+                        "injuries": ["bite-wound", "blood loss"],
```

Comment:
Why remove blood loss as an injury?

Reply:
Because adding it to the list makes it possibly as a random choice, it doesn't add both of the listed injuries.  This means a cat could end up getting just blood loss without a bite-wound or accompanying injury.  Additionally, bite-wounds already have blood loss as a potential paired injury, so just having bite-wound as an option automatically makes blood loss a potential injury for the cat.

Reply:
Ah, mistake fixed! Should be good now